Example/HorizonCalendarExample/HorizonCalendarExample/SwiftUIDayView.swift

```diff
@@ -0,0 +1,81 @@
+// Created by Bryan Keller on 8/23/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import HorizonCalendar
+import SwiftUI
+
+@available(iOS 13.0.0, *)
+struct SwiftUIDayView: View {
+
+  // MARK: Internal
+
+  let dayNumber: Int
+  let isSelected: Bool
+
+  var body: some View {
+    ZStack(alignment: .center) {
+      if #available(iOS 15.0, *) {
+        Circle()
+          .strokeBorder(isSelected ? Color(UIColor.blue) : .clear, lineWidth: 1)
+          .background {
+            Circle()
+              .foregroundColor(
+                isHighlighted
+                ? Color(UIColor.systemFill)
+                : (isSelected ? Color(UIColor.blue.withAlphaComponent(0.15)) : .clear))
+          }
+          .aspectRatio(1, contentMode: .fill)
+      } else {
+        Circle()
+          .foregroundColor(
+            isHighlighted
+            ? Color(UIColor.systemFill)
+            : (isSelected ? Color(UIColor.blue.withAlphaComponent(0.15)) : .clear))
+          .aspectRatio(1, contentMode: .fill)
+      }
+      Text("\(dayNumber)").foregroundColor(Color.black)
+    }
+    // It's unclear how to make this play nicely with `UIScrollView`'s pan gesture recognizer.
+//    .gesture(
+//      DragGesture(minimumDistance: 0)
+//        .updating($isHighlighted) { _, isHighlighted, _ in
+//          isHighlighted = true
+//        })
```

Comment:
Whenever this is recognized, it prevents the scroll view from scrolling the calendar.

Reply:
What if we wrapped this view in a Button so we can specify a highlight state?

Reply:
Oh that seems way better - hopefully that plays nicely with the scroll view. Will try it later today.

Reply:
`Button` doesn't work well since it intercepts taps that would otherwise get passed through to the calendar (which then trigger the `daySelectionHandler` closure). I removed the highlight state code for now, and will explore alternatives later.